ISSUE_TEMPLATE.md

```diff
@@ -1,16 +0,0 @@
-For QUESTIONS, DISCUSSIONS and SUPPORT issues, go to:
```

Comment:
We should keep this? Unless we are de-activating the issues tab altogether...

Reply:
I don't see any reason NOT to de-activate the issues tab; this readme does give specific guidance in the "Help and documentation" section about raising issues in their appropriate repos.

Unfortunately I don't have adequate permissions to disable issue creation. @hsanjuan, can you please do this?